.gitignore

```diff
@@ -16,6 +16,8 @@ pids
 *.pid.lock
 .DS_Store
 
+.env**
```

Comment:
Please remove this. `.env` files should be included, following the Next.js spec for local vs. share variables: https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables

We already include the following lower on in the the `.gitignore`:
```
# dotenv environment variables file
.env.*.local
.env.test
.env.local
```